podpac/core/coordinates/coordinates1d.py

```diff
@@ -56,13 +56,17 @@ def _set_name(self, value):
     # ------------------------------------------------------------------------------------------------------------------
 
     def __repr__(self):
-        return "%s(%s): Bounds[%s, %s], N[%d]" % (
-            self.__class__.__name__,
-            self.name or "?",
-            self.bounds[0],
-            self.bounds[1],
-            self.size,
-        )
+        if self.name is None:
+            name = "%s" % (self.__class__.__name__,)
+        else:
+            name = "%s(%s)" % (self.__class__.__name__, self.name)
+
+        if self.ndim == 1:
+            desc = "Bounds[%s, %s], N[%d]" % (self.bounds[0], self.bounds[1], self.size)
+        else:
+            desc = "Bounds[%s, %s], N[%s], Shape%s" % (self.bounds[0], self.bounds[1], self.size, self.shape)
+
+        return "%s: %s" % (name, desc)
 
     def __eq__(self, other):

```

Comment:
Ah, I'm actually going to fix this. The `ArrayCoordinates1d` and `UniformCoordinates1d` both implement their own `__eq__` method, which calls this one and then makes additional checks. So it is misleading here. These are just the short-cuts in order to avoid comparing coordinates unless necessary.

Reply:
Could you please add a comment to the code to explain this,

Reply:
cfa25b6